src/core/sys/posix/spawn.d

```diff
@@ -0,0 +1,128 @@
+/**
+ * D header file for spawn.h.
+ *
+ * Copyright: Copyright (C) 2018 by The D Language Foundation, All Rights Reserved
+ * Authors:   Petar Kirov
+ * License:   $(LINK2 https://www.boost.org/LICENSE_1_0.txt, Boost License 1.0)
+ * Source:    $(LINK2 https://github.com/dlang/druntime/blob/master/src/core/sys/posix/spawn.d, _spawn.d)
+ * Standards: The Open Group Base Specifications Issue 6, IEEE Std 1003.1, 2004 Edition
+ */
+module core.sys.posix.spawn;
+
+version (Posix):
+public import core.sys.posix.sys.types : mode_t, pid_t;
+public import core.sys.posix.signal : sigset_t;
+public import core.sys.posix.sched : sched_param;
+
+extern(C):
+@nogc:
+nothrow:
```

Comment:
Should this be changed so `nothrow` doesn't apply to `posix_spawn` and `posix_spawnp` below?

Reply:
I think no, because none of the other cancellation points are marked as such in druntime.
Also you can call functions that throw from `nothrow` functions by wrapping them in `try {} catch (Ex) {}`, however you can't do this for glibc cancellation points as the exception they throw can't be caught in D.